extensions/control-plane/api/management-api/policy-definition-api/src/main/java/org/eclipse/edc/connector/controlplane/api/management/policy/v3/PolicyDefinitionApiV3.java

```diff
@@ -119,6 +119,7 @@ record PolicyDefinitionInputSchema(
                         "@context": "http://www.w3.org/ns/odrl.jsonld",
                         "@type": "Set",
                         "uid": "http://example.com/policy:1010",
+                        "profile": "http://example.com/odrl:profile:02",
```

Comment:
this should be added in a `PolicyDefinitionApiV31alpha interface, extended by a dedicated controller

Reply:
done